benchmark/index/multiple_tpch.cpp

```diff
@@ -0,0 +1,294 @@
+// Whether it is a local test with small numbers
```

Comment:
A few high-level things for the first pass:
1. Please replace all these macros to member variables in the class. Part of the coding convention in our system is not to use macros if possible. Macros reduce the readability of the code, and also introduce global states.
2. Please make all the member variables non-static (unless necessary). It's better to isolate as much information in the class instance itself as possible.
3. Do not hard-code the size of arrays (like `tpch_filename_` or `interp_exec_ms_sum_`). Use vectors instead of make them be able to handle variable sizes without the need to recompile the program. You can initialize them in the `SetUp` function if you want.
4. Break the `RandomInsert` benchmark function into smaller blocks or sub-functions:
https://github.com/cmu-db/terrier/wiki/Coding-Guideline#the-wall-of-text
5. Remove unused code.